frontend/packages/rhoas-plugin/src/topology/components/TopologyKafkaPanel.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+import { Alert, AlertActionCloseButton } from '@patternfly/react-core';
+import { Node } from '@patternfly/react-topology';
+import * as UIActions from '@console/internal/actions/ui';
+import './TopologyKafkaPanel.css';
+import { navFactory, SimpleTabNav } from '@console/internal/components/utils';
+import { ResourcesComponent } from './ResourceComponent';
+import { DetailsComponent } from './DetailsComponent';
+
+type PropsFromState = {
+  selectedDetailsTab?: any;
+};
+
+type PropsFromDispatch = {
+  onClickTab?: (name: string) => void;
+};
+
+const stateToProps = ({ UI }): PropsFromState => ({
+  selectedDetailsTab: UI.getIn(['overview', 'selectedDetailsTab']),
+});
+
+const dispatchToProps = (dispatch): PropsFromDispatch => ({
+  onClickTab: (name) => dispatch(UIActions.selectOverviewDetailsTab(name)),
+});
+
+type OwnProps = {
+  item: Node;
+};
+
+type TopologyRhoasPanelProps = PropsFromState & PropsFromDispatch & OwnProps;
+
+export const ConnectedTopologyRhoasPanel: React.FC<TopologyRhoasPanelProps> = ({
+  item,
+  selectedDetailsTab,
+  onClickTab,
+}) => {
+  const [showAlert, setShowAlert] = React.useState(true);
+  const { t } = useTranslation();
+  // Resource
+  const akc = item?.getData().resource;
+  if (!akc) {
+    return <>No data</>;
+  }
+
+  const handleAlertFunction = () => {
+    if (showAlert) {
+      setShowAlert(false);
+    }
+  };
```

Comment:
Redundant check since this function is only called if `showAlert` is truthy.
```suggestion
  const handleAlertFunction = () => {
      setShowAlert(false);
  };
```